src/browser/browser.js

```diff
@@ -51,8 +51,14 @@ define((require, exports, module) => {
 
   const modifier = OS.platform() == 'linux' ? 'alt' : 'accel';
   const KeyDown = KeyBindings({
-    'accel l': _ => Input.Action({action: Focusable.Focus()}),
-    'accel t': _ => WebView.Action({action: WebView.Open()}),
+    'accel l': _ => Input.Action({
+      action: Focusable.Focus(),
+      source: 'keyboard'
```

Comment:
To communicate this information, I'm unsure if we should annotate the action or create a new action type like `Focusable.KeyboardFocus` or `WebView.KeyboardOpen`.